osdep/apple_utils.c

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
i would hate to introduce another apple specific pre-/subfix for files. all other files go with mac(OS)(X), osx or darwin. i personally would prefer macos, since i am planning to unify it at some point.

Reply:
named it this since it applies to iOS and such as well, but sure I can just call it macos if you'd prefer

Reply:
I'm pretty 🤷‍♀️ between `apple` and `darwin`; if anything I _guess_ "apple" is more Technically Correct™ just because the APIs in question have nothing to do with the darwin kernel (which you could in principle run some other userspace with), but eh, not really relevant to actual usage…

Reply:
okay, then we will keep it as apple, i guess.